torchtune/modules/peft/peft_utils.py

```diff
@@ -256,3 +256,11 @@ def get_merged_lora_ckpt(
         del state_dict[f"{module}.lora_a.weight"]
         del state_dict[f"{module}.lora_b.weight"]
     return state_dict
+
+
+def disable_adapter(model: nn.Module, disabled: bool) -> None:
```

Comment:
would be great to add a unit test for this and make sure lora weights are indeed not being used

Reply:
> would be great to add a unit test for this and make sure lora weights are indeed not being used

I conducted the test manually by loading the original base model (without LoRA configuration) and subsequently loaded a LoRA model. Then, I enabled disable_adapter to the LoRA model and verified that the outputs matched. Should I include this logic in the unit test?

Reply:
You can replicate this with a single LoRA linear layer [initialized with constants](https://github.com/pytorch/torchtune/blob/2ab2721e6a176c4a698da9eec1be253d14f908d3/tests/test_utils.py#L61) so it's easier to write up, but yes that sounds like a good unit test

Reply:
Could you please review the addition I made [here](https://github.com/yechenzhi/torchtune/blob/e84729793cef772dc33eab25032144912fbf5441/tests/torchtune/modules/peft/test_peft_utils.py#L408)? I don't often engage in writing unit tests.